flair/trainers/language_model_trainer.py

```diff
@@ -285,12 +296,12 @@ def train(
                     self.model.train()
 
                     # reset variables
-                    hidden = self.model.init_hidden(mini_batch_size)
+                    hidden = self.model.module.init_hidden(mini_batch_size)
```

Comment:
Have you tried running it on single machine - multiple GPUs? I get an error, obviously LightningLite wraps DistributedDataParallel which wraps the LanguageModel class - so this line should be `self.model.module.module.init_hidden`. However this looks not correct to me. When using just CPU, this configurations works. Can you check?

Reply:
thanks for checking and reporting this @whoisjones, I let me test this on multiple GPUs and fix it.